### ISSUE TITLE
Checkboxes/Radio "Other" option

### ISSUE THREAD

**Issue by jaydensmith** [verbb/formie#1875]
### What are you trying to do?

I'd like to enable an option for Checkboxes and Radio fields to allow the frontend user to select an "Other" option which will show a text input.

### What's your proposed solution?

- Add an option to Checkboxes and Radio field options to enable the other option
- Maybe also the ability to set the label for the other option

**Comment by engram-design** [comment]
Yeah, this has cropped up before and is extremely common. I actually thought this was already on our roadmap, but looks like it isn't. Definitely something we'll implement.

Storage is one thing to consider for this, as the provided text won't be one that's listed in the options, and will fail validation, so this _may_ be a breaking change, but we'll see!

**Comment by engram-design** [comment]
Added in [4.0.0-beta.10](https://github.com/verbb/formie/releases/tag/4.0.0-beta.10).
